Makefile

```diff
@@ -18,6 +20,7 @@ b2d_iso: iso_creation
 
 .PHONY: iso_creation
 iso_creation: init
+	@echo "BUILD_ID=${GITTAG}-${DATE}" >> ${CODE_DIR}/os-release
```

Comment:
You can add a variable for BUILD_ID and then use `envsetst` for this also same as version.